tests/CMakeLists.txt

```diff
@@ -3,6 +3,7 @@
 
 add_subdirectory(unit)
 add_subdirectory(end_to_end)
+add_subdirectory(integration)
```

Comment:
Consider gating this behind a cmake option, so that it's easy to disable when packaging. Generally when packaging the build system doesn't have access to the Internet, so `ExternalProject` wouldn't work out of the box.

Reply:
The `ExternalProject` target are not part of the default build, meaning their will be no internet access unless `make nftables` or `make iptables` is called. I tried a local only build and there was no issue.